### PR TITLE
allow init containers in keto

### PR DESCRIPTION
**Pull request by waldner**
Allow specifying init containers for keto.

- [X] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [X] I have read the [security policy](../security/policy).
- [X] I confirm that this pull request does not address a security vulnerability. If this pull request addresses a security vulnerability, I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the
      maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works: no, but I tested it in private tests
- [X] I have added necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: helm/charts/keto/templates/deployment.yaml [hunk 1]
```diff
@@ -52,6 +52,12 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if .Values.deployment.extraInitContainers}}
```
Comment (Demonsthere) [inline helm/charts/keto/templates/deployment.yaml:55]: Please follow the pattern at the top of the file using ternary operators 😉

File: helm/charts/keto/templates/deployment.yaml [hunk 2]
```diff
@@ -52,6 +52,12 @@ spec:
...
+      initContainers:
+        {{- if .Values.deployment.extraInitContainers }}
```
Comment (Demonsthere) [inline helm/charts/keto/templates/deployment.yaml:57]: The double if here seems redundant

File: helm/charts/keto/templates/deployment.yaml [hunk 3]
```diff
@@ -52,6 +52,12 @@ spec:
...
+{{ tpl .Values.deployment.extraInitContainers . | indent 8 }}
```
Comment (Demonsthere) [inline helm/charts/keto/templates/deployment.yaml:58]:
```suggestion
{{- tpl .Values.deployment.extraInitContainers . | nindent 8 }}
```
we prefer to use `nindent` as it allows for indentation of the functions within templates :)

**Review by Demonsthere — CHANGES_REQUESTED** [review]
Hi there! Thanks for the PR, noticing some cosmetic changes 😉

File: helm/charts/keto/templates/deployment.yaml (hunk 3 above)
Comment (waldner) [reply helm/charts/keto/templates/deployment.yaml:58]: Ah well, I copied everything from kratos....but ok, I'll do it.

File: helm/charts/keto/templates/deployment.yaml (hunk 1 above)
Comment (waldner) [reply helm/charts/keto/templates/deployment.yaml]: Done.

File: helm/charts/keto/templates/deployment.yaml (hunk 3 above)
Comment (waldner) [reply helm/charts/keto/templates/deployment.yaml]: Done, I also fixed the equivalent `$extraContainers` later in the file.

**Review by Demonsthere — APPROVED** [review]
Great, thanks! :)

**Comment by waldner** [comment]
Thanks to you! Will it make it to a release soon-ish? We'd like to stop maintaining our own fork....

**Comment by Demonsthere** [comment]
I was planning to make a release once https://github.com/ory/k8s/pull/441 gets resolved, if it won't till end of week, i will make a release on friday

**Pull request merged**